app/navigation/InformationNavigator.scala

```diff
@@ -38,12 +38,23 @@ class InformationNavigator @Inject()() extends BaseNavigator {
     case ChangeTypePage => (userAnswers: UserAnswers) =>
       userAnswers.get(ChangeTypePage) match {
           case Some(Consignee) =>
-            //TODO: Redirect to COD-02
-            testOnly.controllers.routes.UnderConstructionController.onPageLoad()
+            controllers.sections.info.routes.ChangeDestinationTypeController.onPageLoad(userAnswers.ern, userAnswers.arc)
           case _ =>
             controllers.routes.DraftMovementController.onPageLoad(request.ern, request.arc)
         }
 
+    case ChangeDestinationTypePage =>
+      (userAnswers: UserAnswers) => userAnswers.get(ChangeDestinationTypePage) match {
+        case Some(wantsToChangeDestinationType) =>
+          if(wantsToChangeDestinationType) {
+            //TODO: redirect to COD-03
+            testOnly.controllers.routes.UnderConstructionController.onPageLoad()
```

Comment:
The page already exists. It's the `DestinationTypeController`. Mike is going to refactor it to make sure it matches the CoD content and logic but 90% of it is probably already there.

Reply:
Nice, will change